openid/example/main.go

```diff
@@ -22,9 +23,10 @@ import (
 )
 
 var (
-	optional = flag.String("optional", "", "comma separated list of optional simple registration fields.")
-	required = flag.String("required", "", "comma separated list of required simple registration fields.")
-	teams    = flag.String("teams", "", "comma separated list of teams to request membership of.")
+	caveatID = flag.String("caveatid", "", "`caveat ID` of a third-party caveat addressed to login.ubuntu.com")
+	optional = flag.String("optional", "", "comma separated list of optional simple registration `fields`.")
```

Comment:
What's with the backquotes here and elsewhere? If it's referencing some literal things somewhere, maybe the help should make that clear?

Reply:
It changes the way PrintDefaults displays the options.